drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1115,6 +1115,7 @@ function DragAndDropBlock(runtime, element, configuration) {
 
             if (attemptsRemain()) {
                 data.misplaced_items.forEach(function(misplaced_item_id) {
+                    state.last_action_correct = false
```

Comment:
@haikuginger Hm, I see why this would work, but it feels kind of hacky. It would be good if `data.correct` were the only "source of truth" regarding the correctness of a solution (in other words, after setting

```js
state.last_action_correct = data.correct;
```

in L1114 above, ideally it shouldn't be necessary to modify the value of `state.last_action_correct` again). I haven't looked at #96 in detail, but it seems like after that PR, `data.correct` should not be truthy if a solution placed one or more decoy items on the board. Is that not the case?